api/openapi.yaml

```diff
@@ -4,9 +4,6 @@ info:
   title: VMClarity APIs
   version: 1.0.0
 
-servers:
-  - url: /api
-
```

Comment:
why it was removed?

Reply:
The API is no longer served on a sub-path because its its own dedicated microservice there is no need to differentiate. I.e. the API internally is served like: `apiserver:8888/scans` but when accessing it from outside through the API gateway, the gateway adds the subpath like `vmclarity-control-plane/api/scans.